VideoLocker/res/layout/discussion_user_profile_row.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/discussion_user_profile_row"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_marginBottom="@dimen/discussion_responses_standard_margin"
+    android:orientation="horizontal"
+    tools:showIn="@layout/row_discussion_comments_comment">
+
+    <de.hdodenhof.circleimageview.CircleImageView
+        android:id="@+id/profile_image"
+        style="@style/discussion_profile_image"
+        android:layout_width="@dimen/discussion_image_diameter"
+        android:layout_height="@dimen/discussion_image_diameter"
+        android:layout_marginEnd="@dimen/discussion_responses_standard_margin"
+        android:layout_marginRight="@dimen/discussion_responses_standard_margin"
+        android:src="@drawable/xsie" />
+
+    <LinearLayout
+        android:layout_width="wrap_content"
+        android:layout_height="match_parent"
+        android:gravity="center_vertical"
+        android:orientation="vertical">
+
+        <TextView
+            android:id="@+id/discussion_author_text_view"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textColor="@color/edx_grayscale_neutral_base"
+            android:textDirection="locale"
```

Comment:
I just moved this `TextView` from multiple layouts to this independent layout, so, the attribute might've been helpful in other contexts. Removing it for now.